vars-ulm.cfg

```diff
@@ -16,142 +16,142 @@ inverse_flattening = 298.257223563
 # Of variable attributes, which are required, link to VIC docs, duplicative (but required by the CF spec)
 
 [Evap]
-units = kg/m^2s
+units = kg.m-2.s-1
 long_name = total net evaporation from surface and vegetation
 description = Total net evaporation from surface and vegetation
 standard_name = total_net_evaporation
 
 [Qs]
-units = kg/m^2s
+units = kg.m-2.s-1
 long_name = surface runoff
 description = Direct surface runoff
 standard_name = surface_runoff
 
 [Qsb]
-units = kg/m^2s
+units = kg.m-2.s-1
 long_name = baseflow runoff
 description = Baseflow (return flow)
 standard_name = baseflow_runoff
 
 [Qsm]
-units = kg/m^2s
+units = kg.m-2.s-1
 long_name = snow melt
 description = Snow melt
 standard_name = snow_melt
 
 [RootMoist]
-units = kg/m^2
-long_name =  SOIL MOISTURE IN ROOT ZONE relative to SMCWLT
-description = UNTRUSTED because >1000 and <0
+units = kg.m-2
+long_name =  soil moisture in root zone relative to SMCWLT
+description = UNTRUSTED because >1000 and <0 ## Unclear what this means
 standard_name = root_zone_soil_moisture
 
 [SoilMoist]
-units = kg/m^2
-long_name = soil moisture by layer
-description = 
+units = kg.m-2
+long_name = soil moisture per layer
+description = Total soil moisture content per layer
 standard_name = soil_moisture_per_layer
 
 [SMLiqFrac]
 units = -
-long_name = fraction of soil moisture that is liquid, for each soil layer
-description = 
+long_name = fraction of soil moisture that is liquid per soil layer
+description = Fraction of soil moisture that is liquid per soil layer
 standard_name = soil_moisture_liquid_fraction_per_layer
 
 [SoilTemp]
 units = K
-long_name = soil temperature by layer
-description = 
+long_name = soil temperature per layer
+description = Soil temperature per layer
 standard_name = soil_temperature_per_layer
 
 [SWE]
-units = kg/m^2
+units = kg.m-2
 long_name = snow water equivalent
 description = Snow water equivalent
 standard_name = snow_water_equivalent
 
 [SWEVeg]
-units = kg/m^2
+units = kg.m-2
 long_name = snow water equivalent in canopy
-description = 
+description =
 standard_name = snow_water_equivalent_vegetation_canopy
 
 [Qg]
-units = w/m^2
-long_name = SOIL HEAT FLUX (W M-2: NEGATIVE IF DOWNWARD FROM SURFACE)
-description = 
+units = W.m-2
+long_name = soil heat flux
+description = Soil heat flux (negative if downward from surface)
 standard_name = soil_heat_flux
 
 [Qle]
-units = w/m^2
-long_name = ACTUAL LATENT HEAT FLUX (W M-2: NEGATIVE, IF UP FROM SURFACE)
-description = 
+units = W.m-2
```

Comment:
Note that unit names are case sensitive, so `W` rather than `w` to indicate Watts (see udunits)